models/zerion/arbitrum/zerion_arbitrum_trades.sql

```diff
@@ -0,0 +1,103 @@
+{{ config(
+    alias = 'trades'
+    ,partition_by = ['block_date']
+    ,materialized = 'incremental'
+    ,file_format = 'delta'
+    ,incremental_strategy = 'merge'
+    ,unique_key = ['blockchain', 'tx_hash', 'evt_index']
+    )
+}}
+
+{% set project_start_date = '2021-06-01' %}
+{% set zerion_native_currency_address = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' %}
+{% set native_wrapped_currency_address = '0x82af49447d8a07e3bd95bd0d56f35241523fbab1' %}
+{% set native_currency_symbol = 'ETH' %}
+
+WITH zerion_trades AS (
+    SELECT swap.evt_block_time AS block_time
+    , date_trunc('day', swap.evt_block_time) AS block_date
+    , swap.evt_block_number AS block_number
+    , swap.sender AS trader
+    , CASE WHEN swap.inputToken='{{zerion_native_currency_address}}' THEN '{{native_wrapped_currency_address}}'
+        ELSE swap.inputToken
+        END AS token_sold_address
+    , CASE WHEN swap.inputToken='{{zerion_native_currency_address}}' THEN '{{native_currency_symbol}}'
+        ELSE tok_sold.symbol
+        END AS token_sold_symbol
+    , swap.absoluteInputAmount AS token_sold_amount_raw
+    , tok_sold.decimals AS tok_sold_decimals
+    , CASE WHEN swap.outputToken='{{zerion_native_currency_address}}' THEN '{{native_wrapped_currency_address}}'
+        ELSE swap.outputToken
+        END AS token_bought_address
+    , CASE WHEN swap.outputToken='{{zerion_native_currency_address}}' THEN '{{native_currency_symbol}}'
+        ELSE tok_bought.symbol
+        END AS token_bought_symbol
+    , swap.absoluteOutputAmount AS token_bought_amount_raw
+    , tok_bought.decimals AS tok_bought_decimals
+    , pt.from AS tx_from
+    , pt.to AS tx_to
+    , swap.evt_tx_hash AS tx_hash
+    , swap.contract_address
+    , swap.evt_index
+    , swap.marketplaceFeeAmount AS marketplace_fee_amount_raw
+    , swap.protocolFeeAmount AS zerion_fee_amount_raw
+    FROM {{ source('zerion_arbitrum', 'Router_evt_Executed') }} swap
+    INNER JOIN {{ source('arbitrum','transactions') }} pt ON pt.block_number=swap.evt_block_number
```

Comment:
can you add the `is incremental` and `is not incremental` filters in the join conditions to base tables too, similar to prices below?

Reply:
Done!